Safewaan/HonkedIn
# C45choi/create resources

## Trello ticket link

[[Ticket Name](https://trello.com/b/RCwqUZD8/msci-342-scrum-board )
](https://trello.com/c/jLsgYeHg/44-as-a-user-i-can-add-links-to-the-resources-page-so-that-other-users-can-access-resources-that-i-think-are-helpful)


## Implementation description
* Added the create resources page under resources in the navigation bar
* The fields for create resources is the title and the link 



## Steps to test
1. Ensure that the routing to create resources works
2. Ensure that the addition of your resource title and link is inserted into the SQL table, resources
3. Ensure there is a confirmation message
4. Ensure the fields cannot be empty



## What should reviewers focus on?
* Are there any other fields that I should add to this?
